starter-code/css/main.css

```diff
@@ -6,3 +6,133 @@ Green: #00B172
 White: #FFF
 
 */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Helvetica,Arial,sans-serif;
+}
+
+.nav-bar {
+  background-color: white;
+  height: 75px;
+  width: 100%;
+  padding: 15px 10px 5px 10px;
+
+}
+ .nav-bar > ul {
+    width: 1200px;
+    margin: 0 auto;
+ }
+
+ .nav-bar li {
+   text-align: right;
+   float: right;
+   padding: 20px 10px 14px 10px;
+   list-style-type: none;
+   font-size: 20px; 
+   margin:0 0 10px 0;  
+   font-size: 20px;
+   font-weight: 100; 
+ }
+
+ .nav-bar a {
+   text-decoration: none;
+   color: black;
+ }
+
+ .music {
+   text-align: center;
+   height: 800px;
+   background: url("../images/landing.jpg");
+   background-size: cover;
+   background-repeat: no-repeat;  
+   color: white;
+   text-align: center;
+ }
+
+ .music > h1 {
+   position: relative;
+   padding-top: 17%;
+   padding-bottom: 20px;
+   line-height: 70px;
```

Comment:
That is a really large line height, dont do line height in px, people tend to just put something like `line-height: 1.2` without any ending units attached.